### PR TITLE
Renamed aws::AWSConfig -> aws_config::AWSConfig

### PR DESCRIPTION
**Pull request by mattBrzezinski**
Was going a bit fast and didn't realize this was quite the poor name for a kwarg.

### CODE REVIEW

File: src/AWS.jl [hunk 1]
```diff
@@ -486,7 +486,7 @@ Perform a RestXML request to AWS.
 """
 function (service::RestXMLService)(
     request_method::String, request_uri::String, args::AbstractDict{String, <:Any}=Dict{String, String}(); 
-    aws::AWSConfig=aws_config
+    aws_config::AWSConfig=aws_config
```
Comment (omus) [inline src/AWS.jl:489]:
```suggestion
    aws_config::AWSConfig=AWS.aws_config,
```

File: src/AWS.jl [hunk 2]
```diff
@@ -545,7 +545,7 @@ Perform a Query request to AWS.
 """
 function (service::QueryService)(
     operation::String, args::AbstractDict{String, <:Any}=Dict{String, Any}(); 
-    aws::AWSConfig=aws_config
+    aws_config::AWSConfig=aws_config
```
Comment (omus) [inline src/AWS.jl:548]:
```suggestion
    aws_config::AWSConfig=AWS.aws_config,
```

File: src/AWS.jl [hunk 3]
```diff
@@ -592,7 +592,7 @@ Perform a JSON request to AWS.
 """
 function (service::JSONService)(
     operation::String, args::AbstractDict{String, <:Any}=Dict{String, Any}();
-    aws::AWSConfig=aws_config
+    aws_config::AWSConfig=aws_config
```
Comment (omus) [inline src/AWS.jl:595]:
```suggestion
    aws_config::AWSConfig=AWS.aws_config,
```

File: src/AWS.jl [hunk 4]
```diff
@@ -638,7 +638,7 @@ Perform a RestJSON request to AWS.
 """
 function (service::RestJSONService)(
     request_method::String, request_uri::String, args::AbstractDict{String, <:Any}=Dict{String, String}();
-    aws::AWSConfig=aws_config
+    aws_config::AWSConfig=aws_config
```
Comment (omus) [inline src/AWS.jl:641]:
```suggestion
    aws_config::AWSConfig=AWS.aws_config,
```

File: src/AWS.jl (hunk 1 above)
Comment (iamed2) [reply src/AWS.jl:489]: Alternatively, 
```suggestion
    aws_config::AWSConfig=global_aws_config()
```

Comment (omus) [reply src/AWS.jl:489]: I'm also good with that.

**Pull request merged**